crowd_anki/anki/overrides/uuid_operations.py

```diff
@@ -34,7 +34,7 @@ def get_deck_by_uuid(self, uuid):
 
 # Deck configuration
 def get_deck_configuration_by_uuid(self, uuid):
-    return get_from_dict_by_uuid(self, "dconf", uuid)
+    return get_from_dict_by_uuid(self, "decks", uuid)
```

Comment:
I don't think it's the right one. if you see above - we already query this field for decks, here we need deck config

Reply:
Couldn't see what was supposed to be there before, just saw a CrowdAnki_UUID right there next to where it was trying to fetch when stopping with a breakpoint :man_shrugging: tried it out of desperation, which is why I labelled mine as a hotfix :sweat_smile: